src/lib/components/Spreadsheet/Spreadsheet.tsx

```diff
@@ -1,30 +1,71 @@
-import React, { useState, forwardRef, useImperativeHandle, useEffect } from "react";
-import { ReactGrid, Column, Row, Id, CellChange, CellLocation, CellStyle } from '@silevis/reactgrid';
+import React, {
+  useState,
+  forwardRef,
+  useImperativeHandle,
+  useEffect,
+  useRef,
+} from "react";
+import {
+  ReactGrid,
+  Column,
+  Row,
+  Id,
+  CellChange,
+  CellLocation,
+  CellStyle,
+} from "@silevis/reactgrid";
 
-import { 
+import {
   TextCreatableCellTemplate,
   ModifiedDropdownCellTemplate,
-  ButtonCell, ButtonCellTemplate,
-  SpreadsheetHeaderTemplate, SpreadsheetHeaderCell,
-} from './templates';
+  ButtonCell,
+  ButtonCellTemplate,
+  SpreadsheetHeaderTemplate,
+  SpreadsheetHeaderCell,
+} from "./templates";
 import {
-  SpreadSheetProps, SpreadSheetOption,
-  SpreadSheetRow, SpreadSheetCellTypes, SpreadSheetColumnOption,
-  DataRow, DataRowValue, DataColumnHeaderMap, DataColumnId,
+  SpreadSheetProps,
+  SpreadSheetOption,
+  SpreadSheetRow,
+  SpreadSheetCellTypes,
+  SpreadSheetColumnOption,
+  DataRow,
+  DataRowValue,
+  DataColumnHeaderMap,
+  DataColumnId,
   RowAction,
   ColumnValuesMap,
-  CellState, CompositeCellValidation,
-  OptionItem, ValidationReport,
-  StyleState, SpreadsheetCellStyle, StyleStateNote,
-} from './Spreadsheet.types';
-import { isRangePattern, isColonPattern } from '../../helpers';
-import { getParser, replaceVariable } from './formulaParser';
+  CellState,
+  CompositeCellValidation,
+  OptionItem,
+  ValidationReport,
+  StyleState,
+  SpreadsheetCellStyle,
+  StyleStateNote,
+} from "./Spreadsheet.types";
+import { isRangePattern, isColonPattern } from "../../helpers";
+import { getParser, replaceVariable } from "./formulaParser";
+import useClickOutside from "../../hooks/useClickOutside";
+import useOnScreen from "../../hooks/useOnScreen";
+import "./Spreadsheet.css";
```

Comment:
due to build script, place all css lines on `src/styles.css`
add just add new lines

Reply:
then remove line 50